lib/internal/quic/core.js

```diff
@@ -1307,15 +1339,15 @@ class QuicStream extends Duplex {
       ...options,
       allowHalfOpen: true,
       decodeStrings: true,
-      emitClose: true
+      emitClose: true,
+      autoDestroy: true,
     });
     handle.onread = onStreamRead;
     handle[owner_symbol] = this;
     this[async_id_symbol] = handle.getAsyncId();
     this[kHandle] = handle;
     this.#id = id;
     this.#session = session;
-    this._readableState.readingMore = true;
   }
 
   get serverInitiated() {
```

Comment:
I recognize this is not changed (yet) for this PR, but this should strictly return a boolean (same for `unidirectional`).